src/worker/lambda/handlerPuller.go

```diff
@@ -33,6 +33,62 @@ type CacheEntry struct {
 	path    string // where code is extracted to a dir
 }
 
+func ccopy(src, dest string) error {
```

Comment:
not sure what the "c" stands for -- can we give it a longer name?

Reply:
Was thinking it to represent "customeBuiledCopy". Changed the name to "Copy".